notification2-examples/hello-world-microservice/demo/device.py

```diff
@@ -14,14 +14,13 @@
 
 
 def get_session(tenant_id=_tenant_id, username=_username, password=_password):
-    auth_string = f'{tenant_id}/{username}:{password}'
     session = Session()
     session.verify = True
     session.headers.update({
         'Content-Type': 'application/json',
-        'Accept': 'application/json',
-        'Authorization': 'Basic ' + base64.b64encode(auth_string.encode()).decode()
+        'Accept': 'application/json'
     })
+    session.auth = (username, password)
```

Comment:
question: Is the tenant ID required as part of the username? Maybe it is not required if using the tenant's own URL if we are doing that. Does this differ (it might for us locally only) and if so should we comment on it?

Reply:
Done.

Reply:
Sorry, is that thumbs-up for the first question or second? I might just be that we need a comment somewhere pointing out that when using tenant-specific URLs are used, don't use tenant-id in username. If non-tenant-specific one is used, provide tenant-id/username.

I think my second question is unclear. What I meant was: 
Does this vary between tenant-specific and non-tenant-specific URL for customers? if so should we comment on it.
It may be that we only see the non-specific case in local-environments (so don't trouble them with it)

Reply:
... sorry again. You've updated before I clicked comment - I'll look again

Reply:
All good - thanks